code/src/components/Thoughts.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useEffect } from 'react';
+import moment from 'moment';
+
+const Thoughts = ({ thoughts, setThoughts }) => {
+  // get list of thoughts but only once
```

Comment:
Your code is easy to follow but it's nice to have the comments to get an even better understanding.